### ISSUE TITLE
Can't set apns parameters with google-fcm1

### ISSUE THREAD

**Issue by pognibene** [Byron/google-apis-rs#336]
Hi,
In the generated code, ApnsConfig::payload is an Option<HashMap<String, String>>.
However, according to Apple/fcm specs, this is the payload we should send:
```
"apns": {
    "fcmOptions": null,
    "headers": {
      "apns-priority": "10"
    },
    "payload": {
      "aps": {
           "sound":"sound.wav",
           "alert": {
               "loc-key": "my_loc_key"
           }
      }
    }
  }
```

I can't find a way to generate the proper apns payload with a String value.
Is this a generation bug, or I overlooked a way to achieve the desired result?
Thank you for your help!

**Comment by tyranron** [comment]
@Byron I guess this may be closed now, as should be fixed by #404.

**Comment by Byron** [comment]
Thanks for the hint! Done.
